src/player.cpp

```diff
@@ -11043,7 +11043,7 @@ std::string player::weapname() const
         return _( "fists" );
 
     } else {
-        return weapon.tname();
+        return weapon.tname( false );
```

Comment:
calls `item::tname` with following parameters: `[ quantity = 0, with_prefix = default (false) ]`, is this intended?

Reply:
it was to hide the item bars for weapon on the sidebar. same as the other one you reviewed,
i didn't know then that there was a setting for this.